### ISSUE TITLE
Should we only be having GitHub URL's for leads in the global directory?

### ISSUE THREAD

**Issue by saranshkataria** [fbdevelopercircles/FbDevcCommunityContent#63]
I was under the impression that we were adding GitHub profile links, but it is a mix fo Facebook and GitHub right now. We should be consistent with that in my opinion?

**Comment by BolajiAyodeji** [comment]
I agree with you. Since the official circle's directory already has facebook links of leads, it'll be ideal to use GitHub here.

What do you think?

**Comment by saranshkataria** [comment]
I agree. We have some cleanup to do as well as not approve the pending PR's in that case since they all contain links to Facebook profiles instead.

**Comment by BolajiAyodeji** [comment]
Got it, let's get the optimized README's merged first then, we clean up next and update the contributor's docs.

Sounds good?

**Comment by saranshkataria** [comment]
sounds like a plan

**Comment by BolajiAyodeji** [comment]
You go ahead and clean up the links while I update the docs to show contributors what we expect from them.

Sounds good?

**Comment by saranshkataria** [comment]
Sounds good!
